samples/sampler/stories/default/VideoPlayer.js

```diff
@@ -135,6 +136,7 @@ export const _VideoPlayer = () => {
 			</label>
 			<VideoPlayer
 				autoCloseTimeout={number('autoCloseTimeout', Config, 7000)}
+				backButtonAriaLabel={text('backButtonAriaLabel', Config)}
```

Comment:
I think you missed above
`text('backButtonAriaLabel', _VideoPlayer, Config, 'go to previous');`

https://github.com/enactjs/sandstone/pull/1145/files#diff-17238cddc27f30bf52c486b94901be1f84908c67c7b534aebd9b4e9d952484cc

Reply:
It's fixed.